x-pack/plugins/security_solution/public/common/components/header_page/index.tsx

```diff
@@ -104,13 +105,14 @@ const HeaderPageComponent: React.FC<HeaderPageProps> = ({
   children,
   draggableArguments,
   isLoading,
+  rightSideItems,
   subtitle,
   subtitle2,
   title,
   titleNode,
 }) => (
   <>
-    <EuiPageHeader alignItems="center" bottomBorder={border}>
+    <EuiPageHeader alignItems="center" bottomBorder={border} rightSideItems={rightSideItems}>
```

Comment:
Added this in preparation for adding the take actions dropdown which will be added in a follow up PR